### PR TITLE
ENG-1083 update auto refresh

### PR DESCRIPTION
**Pull request by aletex1994**
• Refactored NFTLibrary and Collection Library, after library update data is automatically refreshed.
• Small fixes to the layout ( progress bar was not showed in Collection Update, Web Update)

### CODE REVIEW

**Comment by Motokoder** [comment]
Please save all your file with prettier, then I will review. Thanks.

**Review by Motokoder — CHANGES_REQUESTED** [review]
Comments are not resolved.

File: packages/apps/library/src/components/CollectionLibrary/index.tsx [hunk 1]
```diff
@@ -25,51 +26,47 @@ function formatBytes(bytes, decimals = 2) {
 }
 
 export const CollectionLibrary = (props: any) => {
+  const [objLibraryData, setObjLibraryData] = React.useState<SimplifiedMeta>({
```
[the remark this answers is not in the document]
Comment (aletex1994) [reply packages/apps/library/src/components/CollectionLibrary/index.tsx:29]: objLibraryData --> simplifiedLibraryMeta

File: packages/apps/library/src/components/CollectionLibrary/index.tsx [hunk 2]
```diff
@@ -25,51 +26,47 @@ function formatBytes(bytes, decimals = 2) {
...
+    title: '',
+    content_type: '',
+    location: '',
+    location_type: '',
+    size: 0,
+    isMutable: false,
+  });
+
+  const processMetadata = async () => {
+    const metadata = await props.library3;
```
[the remark this answers is not in the document]
Comment (aletex1994) [reply packages/apps/library/src/components/CollectionLibrary/index.tsx]: Deleted await

File: packages/apps/library/src/components/CollectionLibrary/index.tsx [hunk 3]
```diff
@@ -25,51 +26,47 @@ function formatBytes(bytes, decimals = 2) {
...
+    const libraryId = metadata.Class?.filter((res) => res.name === 'library_id')[0].value.Text;
+    const title = metadata.Class?.filter((res) => res.name === 'title')[0].value.Text;
+    const contentType = metadata.Class?.filter((res) => res.name === 'content_type')[0].value.Text;
+    const location = metadata.Class?.filter((res) => res.name === 'location')[0].value.Text;
+    const locationType = metadata.Class?.filter((res) => res.name === 'location_type')[0].value
+      .Text;
+    const size = metadata.Class?.filter((res) => res.name === 'size')[0].value.Nat;
+    let isMutable = false;
+    if (metadata.Class.filter((item) => item.name === 'com.origyn.immutable_library')[0]) {
```
[the remark this answers is not in the document]
Comment (aletex1994) [reply packages/apps/library/src/components/CollectionLibrary/index.tsx:48]: Done, created a folder constants > constants.ts

**Review by Motokoder — APPROVED** [review]

**Pull request merged**